### PR TITLE
[tests-only][full-ci] add test to check move activities of destination file and folder

### PR DESCRIPTION
**Pull request by prashant-gurung899**
## Description
This PR adds tests to check activities of destination file and folder after moving a file into a folder.

``` feature
Scenario: check activities of destination folder after moving a file into it
Scenario: check activities of destinaton file and folder after moving a 0 byte file into it
Scenario: check activities of destinaton file and folder after moving a file by renaming destination file
Scenario: check activities of destinaton file and folder after moving a 0 byte file by renaming destination file
```

## Related Issue

- Part of: https://github.com/owncloud/ocis/issues/9543
- https://github.com/owncloud/ocis/issues/9543#issuecomment-2431703972

## How Has This Been Tested?

- locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/apiActivities/activities.feature [hunk 1]
```diff
@@ -810,6 +810,808 @@ Feature: check activities
       }
       """
 
+
+  Scenario: check activities of destination folder after moving a file into it
```
Comment (saw-jan) [inline tests/acceptance/features/apiActivities/activities.feature:814]: can we check for both here? you can merge the scenarios if there's already one for destination file
```suggestion
  Scenario: check activities of destination file and folder after moving a file into it
```

Comment (prashant-gurung899) [reply tests/acceptance/features/apiActivities/activities.feature:814]: Yeah we can do that :100:

File: tests/acceptance/features/apiActivities/activities.feature [hunk 2]
```diff
@@ -769,11 +879,701 @@ Feature: check activities
...
+                }
+              ]
+            }
+          }
+        }
+      }
+      """
+
+
+  Scenario: check activities of destinaton file and folder after moving a 0 byte file into it
```
Comment (amrita-shrestha) [inline tests/acceptance/features/apiActivities/activities.feature:924]:
```suggestion
  Scenario: check activities of file and folder in new destination after moving a 0 byte file into it
```

File: tests/acceptance/features/apiActivities/activities.feature [hunk 3]
```diff
@@ -769,11 +879,701 @@ Feature: check activities
...
+                }
+              ]
+            }
+          }
+        }
+      }
+      """
+
+
+  Scenario: check activities of destinaton file and folder after moving a file by renaming destination file
```
Comment (amrita-shrestha) [inline tests/acceptance/features/apiActivities/activities.feature:1142]:
```suggestion
  Scenario: check activities of file and folder in new destination (renaming destination file)
```

**Review by saw-jan — APPROVED** [review]

**Pull request merged**